### PR TITLE
add missing config option to XSD file

### PR DESCRIPTION
**Pull request by xabbuh**
This option was introduced in #153.

### CODE REVIEW

**Comment by xabbuh** [comment]
I didn't realise that I have push access here, but I reverted that change in 2f92a81dc580e11fefaad005b82c3e4bb1a1030e. Sorry for that.

**Comment by weaverryan** [comment]
:+1:

File: Resources/config/schema/monolog-1.0.xsd [hunk 1]
```diff
@@ -71,6 +71,7 @@
         <xsd:attribute name="document-type" type="xsd:string" />
         <xsd:attribute name="ignore-error" type="xsd:string" />
         <xsd:attribute name="api_version" type="xsd:string" />
+        <xsd:attribute name="include-stacktraces" type="xsd:boolean" />
```
Comment (stof) [inline Resources/config/schema/monolog-1.0.xsd]: should be a type string to allow using DIC parameters in the config

Comment (xabbuh) [inline Resources/config/schema/monolog-1.0.xsd]: done

**Comment by stof** [comment]
Thank you @xabbuh.

**Pull request closed without merging**